docs/view_components/welcome.rst

```diff
@@ -37,14 +37,17 @@ Requirement
 2. If the user ticks the check-box *Do not show again*, the welcome view-component should not appear to the user during any subsequent launch of the tool
 3. The user could later re-enable the appearance of the pop-up window through *Display* or *Preferences* menu in *Settings*
 
+Link with other views .. isn't this missing here?
```

Comment:
It is missing indeed. However I think it it would be empty. In that case should we leave it empty, or omit it?

Reply:
For me its nicer to have it, but have it empty. Otherwise, i would probably think that something is missing. What do you think?

Reply:
I agree with @srhbrnds that we should have it, but leave it empty, following the template.

Reply:
If the formatting is not perturbed by not having anything written after the field it is ok :)